src/sentiment.py

```diff
@@ -265,6 +268,21 @@ def __init__(self, url=None, frequency=3600):
 
             logger.info("Will get news headlines again in %s sec..." % self.frequency)
             time.sleep(self.frequency)
+    def cleanup(self):
+        new_headline = []
+        new_followlink = []
+        if len(self.headlines) > self.max_cache:
+            for i in range(self.max_cache / 2, len(self.headlines) - 1):
+              new_headline.append(self.headlines[i])
+
+        self.headlines = new_headline
+
+        if len(self.followedlinks) > self.max_cache:
+            for i in range(self.max_cache / 2, len(self.followedlinks) - 1):
```

Comment:
Could this be replaced with `new_headline = self.headlines[self.max_cache / 2, len(self.headlines) - 1);`?